test/source-sink-errors/mzcompose.py

```diff
@@ -77,11 +77,6 @@ def populate(self, c: Composition) -> None:
                   FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY CONNECTION csr_conn
                   ENVELOPE DEBEZIUM
                   /* WITH ( REMOTE 'storaged:2100' ) REMOTE causes the status source to be empty */
```

Comment:
I spent some time investigating this without any luck. `storaged`'s writes are acknowledged by persist, but it seems as though they never become visible to readers.

I'm not very familiar with docker compose... are writes in one container visible to another? [The docs suggest they are not by default](https://docs.docker.com/storage/volumes/#share-data-between-machines); I'm not sure if we do anything custom here.

Reply:
Thanks, will try to figure that out myself.